setup.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 
 try:
-    from setuptools import setup
+    from setuptools import setup, find_packages
```

Comment:
In the case of an exception the symbol `find_packages` isn't defined and would therefore fail later.

Reply:
This was part of debugging, reverting to the old setup.py